assets/checkout.js

```diff
@@ -0,0 +1,116 @@
+/******/ (function(modules) { // webpackBootstrap
+/******/ 	// The module cache
+/******/ 	var installedModules = {};
+/******/
+/******/ 	// The require function
+/******/ 	function __webpack_require__(moduleId) {
+/******/
+/******/ 		// Check if module is in cache
+/******/ 		if(installedModules[moduleId]) {
+/******/ 			return installedModules[moduleId].exports;
+/******/ 		}
+/******/ 		// Create a new module (and put it into the cache)
+/******/ 		var module = installedModules[moduleId] = {
+/******/ 			i: moduleId,
+/******/ 			l: false,
+/******/ 			exports: {}
+/******/ 		};
+/******/
+/******/ 		// Execute the module function
+/******/ 		modules[moduleId].call(module.exports, module, module.exports, __webpack_require__);
```

Comment:
Codacy has a fix for the issue: [Replace `/******/·↹↹modules[moduleId].call(module.exports,·module,·module.exports,·__webpack_require__` with `····/******/·modules[moduleId].call(⏎······module.exports,⏎······module,⏎······module.exports,⏎······__webpack_require__⏎····`](https://app.codacy.com/gh/Anilkum201241/Anilkum201241/pullRequest?prid=10022322)
```suggestion
    /******/ modules[moduleId].call(
```